docs/source/tensors.rst

```diff
@@ -176,492 +160,510 @@ Tensor class reference
-    Tensor.permute
-    Tensor.pin_memory
-    Tensor.pinverse
-    Tensor.polygamma
-    Tensor.polygamma_
-    Tensor.pow
-    Tensor.pow_
-    Tensor.prod
-    Tensor.put_
-    Tensor.qr
-    Tensor.qscheme
-    Tensor.quantile
-    Tensor.nanquantile
-    Tensor.q_scale
-    Tensor.q_zero_point
-    Tensor.q_per_channel_scales
-    Tensor.q_per_channel_zero_points
-    Tensor.q_per_channel_axis
-    Tensor.rad2deg
-    Tensor.random_
-    Tensor.ravel
-    Tensor.reciprocal
-    Tensor.reciprocal_
-    Tensor.record_stream
-    Tensor.register_hook
-    Tensor.remainder
-    Tensor.remainder_
-    Tensor.renorm
-    Tensor.renorm_
-    Tensor.repeat
-    Tensor.repeat_interleave
-    Tensor.requires_grad
-    Tensor.requires_grad_
-    Tensor.reshape
-    Tensor.reshape_as
-    Tensor.resize_
-    Tensor.resize_as_
-    Tensor.retain_grad
-    Tensor.roll
-    Tensor.rot90
-    Tensor.round
-    Tensor.round_
-    Tensor.rsqrt
-    Tensor.rsqrt_
-    Tensor.scatter
-    Tensor.scatter_
-    Tensor.scatter_add_
-    Tensor.scatter_add
-    Tensor.select
-    Tensor.set_
-    Tensor.share_memory_
-    Tensor.short
-    Tensor.sigmoid
-    Tensor.sigmoid_
-    Tensor.sign
-    Tensor.sign_
-    Tensor.signbit
-    Tensor.sgn
-    Tensor.sgn_
-    Tensor.sin
-    Tensor.sin_
-    Tensor.sinc
-    Tensor.sinc_
-    Tensor.sinh
-    Tensor.sinh_
-    Tensor.asinh
-    Tensor.asinh_
-    Tensor.arcsinh
-    Tensor.arcsinh_
-    Tensor.size
-    Tensor.slogdet
-    Tensor.solve
-    Tensor.sort
-    Tensor.split
-    Tensor.sparse_mask
-    Tensor.sparse_dim
-    Tensor.sqrt
-    Tensor.sqrt_
-    Tensor.square
-    Tensor.square_
-    Tensor.squeeze
-    Tensor.squeeze_
-    Tensor.std
-    Tensor.stft
-    Tensor.storage
-    Tensor.storage_offset
-    Tensor.storage_type
-    Tensor.stride
-    Tensor.sub
-    Tensor.sub_
-    Tensor.subtract
-    Tensor.subtract_
-    Tensor.sum
-    Tensor.sum_to_size
-    Tensor.svd
-    Tensor.swapaxes
-    Tensor.swapdims
-    Tensor.symeig
-    Tensor.t
-    Tensor.t_
-    Tensor.tensor_split
-    Tensor.tile
-    Tensor.to
-    Tensor.to_mkldnn
-    Tensor.take
-    Tensor.take_along_dim
-    Tensor.tan
-    Tensor.tan_
-    Tensor.tanh
-    Tensor.tanh_
-    Tensor.atanh
-    Tensor.atanh_
-    Tensor.arctanh
-    Tensor.arctanh_
-    Tensor.tolist
-    Tensor.topk
-    Tensor.to_sparse
-    Tensor.trace
-    Tensor.transpose
-    Tensor.transpose_
-    Tensor.triangular_solve
-    Tensor.tril
-    Tensor.tril_
-    Tensor.triu
-    Tensor.triu_
-    Tensor.true_divide
-    Tensor.true_divide_
-    Tensor.trunc
-    Tensor.trunc_
-    Tensor.type
-    Tensor.type_as
-    Tensor.unbind
-    Tensor.unfold
-    Tensor.uniform_
-    Tensor.unique
-    Tensor.unique_consecutive
-    Tensor.unsqueeze
-    Tensor.unsqueeze_
-    Tensor.values
-    Tensor.var
-    Tensor.vdot
-    Tensor.view
-    Tensor.view_as
-    Tensor.where
-    Tensor.xlogy
-    Tensor.xlogy_
-    Tensor.zero_
+   .. warning::
```

Comment:
Looks like this picked up some changes it shouldn't? I would expect just one new line for cov